### PR TITLE
noise

### PR DESCRIPTION
**Pull request by valoeghese**
noise

### CODE REVIEW

File: bin/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/main/
```
Comment (valoeghese) [inline bin/.gitignore]: irritatered

**Comment by valoeghese** [comment]
noise

**Comment by hYdos** [comment]
noise

**Review by hYdos — APPROVED** [review]

**Review by hYdos — APPROVED** [review]

**Review by hYdos — APPROVED** [review]

**Review by hYdos — APPROVED** [review]

**Review by hYdos — APPROVED** [review]

File: src/main/java/p0nki/pesl/api/builtins/PESLBuiltins.java [hunk 2]
```diff
@@ -110,6 +111,12 @@
                     value = arguments.get(i).asBoolean().getValue();
                 }
                 return new BooleanObject(value);
+            }))
+            .put("noise", FunctionObject.of(false, arguments -> {
+                PESLEvalException.validArgumentListLength(arguments, 1, 2);;
```
Comment (jearmstrong21) [inline src/main/java/p0nki/pesl/api/builtins/PESLBuiltins.java]: Double semicolon :irritatered:

Very bad pr, so many conflicts with other prs and so many problems, trying to sneak in bad code but I hereby block your malicious attempt.

**Review by jearmstrong21 — CHANGES_REQUESTED** [review]

**Comment by valoeghese** [comment]
@jearmstrong21 @jearmstrong21 @jearmstrong21

**Pull request merged**